### DISCUSSION TITLE
Can i get a statistic about past conferention?

### DISCUSSION THREAD

**Discussion by xgerasim** [mynaparrot/plugNmeet-server#517] [Q&A] [answered]
Hi!
Can i get a statistic about past conferention? For example, count of users, duration of conference?
Thanks!

I use this

$roomInfo = new FetchPastRoomsParameters();
$roomInfo->getFrom(0);
$roomInfo->getLimit(30);
$roomInfo->getOrderBy('DESC');
$roomInfo->setRoomIds(array('room01'));

$pastRoomInfo = new PastRoomInfo($roomInfo);

echo '<pre>';
print_r($pastRoomInfo);
echo '</pre>';

I am not a professional, i may a make the mistakes

**Comment by jibon57** [comment]
Hi,

For easy usage you may use: https://github.com/mynaparrot/plugNmeet-sdk-php/blob/f5097454511c81e1ca519d3ecb5345d63f84f902/examples/plugNmeetConnect.php#L469

Some examples can be found in example directory: https://github.com/mynaparrot/plugNmeet-sdk-php/tree/main/examples

**Comment by xgerasim** [comment]
Hi!
Thank you fir last messege!

But i have a few questions.

![image](https://github.com/mynaparrot/plugNmeet-server/assets/149760615/b615f96d-ecb7-4c9d-9d82-d390399efe07)

I did not broadcast why the broadcasts are automatically started and last 5 minutes. And also why the participants are not being displayed, even though they were present at the broadcast. Thanks!

**Reply by xgerasim** [reply]
I'm sorry, I figured out why broadcasts with a duration of 5 minutes are created. But I don't understand why the number of users is not recorded

**Reply by jibon57** [reply, marked as answer]
That's expected because during end of the session there were no one. To get better analytic report you may use: https://www.plugnmeet.org/docs/api/analytics/fetch
